pkg/manager/member/tidb_member_manager.go

```diff
@@ -151,8 +154,12 @@ func (tmm *tidbMemberManager) syncTiDBStatefulSetForTidbCluster(tc *v1alpha1.Tid
 		tc.Status.TiDB.StatefulSet = &apps.StatefulSetStatus{}
 		return nil
 	}
-
-	if !tc.TiKVIsAvailable() {
+	initialized, err := controller.GetPDClient(tmm.pdControl, tc).GetClusterInitialized()
```

Comment:
Doing an API call here may slow down the sync loop. Previously it checked the status. Perhaps this API call should be done elsewhere and set a status field that is checked here?

Reply:
I moved this into the PD status: it will be easier to create a PD API that reduces our API calls there.